ui/StatusQ/src/wallet/managetokenscontroller.cpp

```diff
@@ -112,18 +113,24 @@ void ManageTokensController::showHideGroup(const QString& groupId, bool flag)
     rebuildCommunityTokenGroupsModel();
 }
 
-void ManageTokensController::saveSettings()
+void ManageTokensController::saveSettings(bool reuseCurrent)
 {
     Q_ASSERT(!m_settingsKey.isEmpty());
 
+    setSettingsDirty(true);
```

Comment:
I've spent a lot of trying to figure out this `settingsDirty`. I was expecting to see an update on the settingsDirty after something has changed in the settings (not before) and to be used like any other `dirty` indicator.

As far as I can see it's used as a revision to trigger some qml bindings when it's changing. WDYT of renaming it to something like `settingsRevision`?

Reply:
Yeah, it does exactly that; to retrigger the main view when the settings change. But given the fact that this whole local QSettings stuff is going away in the future anyway... :)